h/views/account_signup.py

```diff
@@ -69,6 +57,27 @@ def post(self):
 
         return {"js_config": self.js_config, "heading": heading, "message": message}
 
+    @exception_view_config(
+        ValidationFailure,
+        request_method="POST",
+        renderer="h:templates/accounts/signup-post.html.jinja2",
+    )
+    def validation_failure(self):
```

Comment:
This pattern align with the example in https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/views.html#custom-exception-views.